mdnsbridge/mdnsbridge.py

```diff
@@ -74,15 +74,23 @@ def _mdns_callback(self, data):
             priority = 0
             versions = ["v1.0"]
             protocol = "http"
-            if "pri" in data["txt"]:
-                if data["txt"]["pri"].isdigit():
-                    priority = int(data["txt"]["pri"])
-            if "api_ver" in data["txt"]:
-                versions = data["txt"]["api_ver"].split(",")
-            if "api_proto" in data["txt"]:
-                protocol = data["txt"]["api_proto"]
+            txt_data = {}
+            # Convert txt data from binary data in python3
+            for key, value in data["txt"].items():
+                if type(key) is not str:
+                    key = key.decode('ascii')
+                if type(value) is not str:
+                    value = value.decode('ascii')
+                txt_data[key] = value
```

Comment:
This line is interesting. If `text_data[b'key']` already exists and you set `txt_data['key']` to a value, can you end up with a dict like the following, or does it resolve itself to the string version to avoid duplicates? `{b'key': b'value', 'key': 'value'}